transcom/mymove
# B19774 MAIN 20240627

## Agility ticket
[B19774](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19774)

## INT PR
[13111](https://github.com/transcom/mymove/pull/13111)


## Summary
When a service counselor is reviewing a PPM, a change was need to ensure Estimated SIT Weight does not exceed Estimated PPM weight.

### How to test
* Customer:
  - Create a new move with a PPM Shipment.  
  - On the PPM shipment page, set "Storage" to "Yes".
  - On the Estimated Weight page, enter 1,000 lbs.
* SC:
  - Open the move
  - On the Move Details page, click "Edit Shipment".
  - Scroll down to "Storage In Transit (SIT)" section
  - For "Estimated SIT Weight", enter 1,500 lbs.
  - An error message should appear:  "Enter a weight no greater than the shipment's estimated weight"

## Screenshots
![image](https://github.com/transcom/mymove/assets/146897935/951e4f26-d35f-46fa-b1bf-e0ff66001f23)
